tls/s2n_connection.h

```diff
@@ -340,7 +340,7 @@ struct s2n_connection {
 
     /* Bitmap to represent preferred list of keyshare for client to generate and send keyshares in the ClientHello message.
      * The least significant bit (lsb), if set, indicates that the client must send an empty keyshare list.
-     * Each bit value in the bitmap indiciates the corresponding curve in the ecc_preferences list for which a key share needs to be generated.
+     * Each bit value in the bitmap indicates the corresponding curve in the ecc_preferences list for which a key share needs to be generated.
```

Comment:
You don't otherwise touch this file. Is it worth fixing this as part of this PR?

Reply:
Why not? It's a simple spell fix.

Reply:
I mean, there are probably misspellings in many files xD It just complicates the diff to correct unrelated spelling mistakes.